README.md

```diff
@@ -269,13 +269,13 @@ To run the test suite, you first need to clone this repo and then install all
 dependencies [through Composer](https://getcomposer.org):
 
 ```bash
-$ composer install
+composer install
 ```
 
 To run the test suite, go to the project root and run:
 
 ```bash
-$ php vendor/bin/phpunit
+php vendor/bin/phpunit
```

Comment:
Looks like a good opportunity to also get rid of the `php` binary here for consistency reasons (not needed and also not used in most components) :+1: